plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/jsonrpc/DefaultGLSPServer.java

```diff
@@ -99,24 +99,37 @@ public void connect(final GLSPJsonrpcClient clientProxy) {
    public void process(final ActionMessage message) {
       log.debug("process " + message);
       String clientId = message.getClientId();
+
+      Function<Throwable, Void> errorHandler = ex -> {
```

Comment:
Please extract into dedicated method that provides the errorHandler. This way subclasses can easily use a customized errorHandler